src/inmanta/protocol/methods_v2.py

```diff
@@ -52,6 +52,7 @@ def put_partial(
     resource_sets: Optional[dict[ResourceIdStr, Optional[str]]] = None,
     removed_resource_sets: Optional[list[str]] = None,
     pip_config: Optional[PipConfig] = None,
+    bypass_base_version_check: bool = False,
```

Comment:
This one is end-user exposed as well, so same name and help remark as on the CLI.